R/fda-table_04.R

```diff
@@ -0,0 +1,102 @@
+#' FDA Table 4: Patient Disposition, Pooled Analyses
+#'
+#' @details
+#' * `df` must contain `SAFFL`, `USUBJID`, and the variables specified by `arm_var`
+#'   and `trtedtm_var`.
+#' * If specified, `alt_counts_df` must contain `SAFFL`, `USUBJID`, and the variable specified by `arm_var`.
+#' * Flag variables (i.e. `XXXFL`) are expected to have two levels: `"Y"` (true) and `"N"` (false). Missing values in
+#'   flag variables are treated as `"N"`.
+#' * Columns are split by arm. Overall population column is excluded by default (see `lbl_overall` argument).
+#' * Numbers in table "Patients Treated" section are the absolute numbers of patients and fraction of `N`.
+#' * All-zero rows are not removed by default (see `prune_0` argument).
+#' * Records with missing treatment start and/or end datetime are excluded from all calculations.
+#'
+#' @inheritParams argument_convention
+
+#' @examples
```

Comment:
Could you please double check whether this example works on your end? I tried to run it, but it throws an error:
![image](https://github.com/pharmaverse/falcon/assets/85986258/c2559fda-a32e-4a22-bee6-67716499630e)

Reply:
FYI, I withdraw my assignment to act as a reviewer since I will be ooo until July 24th.

Reply:
Hi Jessica, I have updated the function for table 4 and it seems there is no errors like that. Thank you

Reply:
It is only the example in the roxygen documentation that does not work. The problem is that your example adsl does not have columns "RANDFL", "PPROTFL", and "DCTREAS", which causes the function to stop. So if you provide the code to create dummy columns for these variables, it should work.

Reply:
Hi Jessica, the new fda-table_04 has been updated to our 57 branch, please take a look when available.